SimpleProducer.py

```diff
@@ -103,13 +113,24 @@ def cleanup_processes(procs):
 def cleanup_producer(producer):
     producer.close()
 
+def print_data_results(keys, dict_key):
+    print(dict_key + " -", end=' ')
+    for key in keys:
+        print(key + " -", end=' ')
+        length = sum(1 for x in (item[key] for item in shared_dict[dict_key]))
+        print("Mean: " + str(sum(item[key] for item in shared_dict[dict_key]) / length), end=' ')
+        print("Max: " + str(max(item[key] for item in shared_dict[dict_key])), end=' ')
+        print("Min: " + str(min(item[key] for item in shared_dict[dict_key]))) 
+        
 def produce_output(dict_key, output_time):
     if len(shared_dict[dict_key]) == 0:
         return
-    print(dict_key + " - Mean: " + str(sum(shared_dict[dict_key]) / len(shared_dict[dict_key])) + " Max: " + str(max(shared_dict[dict_key])) + " Min: " + str(min(shared_dict[dict_key])) ) 
+    keys = shared_dict[dict_key][0].keys()
+    print_data_results(keys, dict_key)
     with open("output-send-" + str(int(output_time)) + ".csv", 'a', newline='') as output_file:
```

Comment:
Can you put the out file in the out/ directory?

Reply:
Sure thing